uw-frame-components/portal/settings/controllers.js

```diff
@@ -3,6 +3,21 @@
 define(['angular'], function(angular) {
   return angular.module('portal.settings.controllers', [])
 
+    .filter('prettyName',function() {
```

Comment:
Suggest a comment hinting at what this filter does. Suggest re-name to better reflect what the filter does or is intended to do.  I think this converts ugly technical theme identifiers into attractive user-facing names for themes.